samples/ws2022s/provisioners/chef/cookbooks/gusztavvargadr_packer_w/recipes/cleanup.rb

```diff
@@ -0,0 +1,43 @@
+# powershell_script 'Execute NGen' do
+#   code <<-EOH
+#     Get-ChildItem $env:SystemRoot\\Microsoft.net\\NGen.exe -recurse | %{ & $_ executeQueuedItems }
+#   EOH
+#   action :run
+# end
+
+powershell_script 'Clearing temporary files' do
+  code <<-EOH
+    @(
+        "$env:localappdata\\temp\\*",
+        "$env:windir\\logs",
+        "$env:windir\\panther",
+        "$env:programdata\\Microsoft\\Windows Defender\\Scans\\*"
+        ) | % {
+          Write-Host "Removing $_"
+          try {
+            Takeown /d Y /R /f $_
+            Icacls $_ /GRANT:r administrators:F /T /c /q  2>&1 | Out-Null
+            Remove-Item $_ -Recurse -Force | Out-Null
+          } catch { $global:error.RemoveAt(0) }
+        }
+  EOH
+  action :run
+end
+
+gusztavvargadr_windows_updates '' do
+  action :cleanup
+end
+
+powershell_script 'Optimizing volume' do
+  code <<-EOH
+    Optimize-Volume -DriveLetter C -Analyze -Defrag
+  EOH
+  action :run
+end
+
+powershell_script 'Zeroing volume' do
+  code <<-EOH
+    sdelete -nobanner -z C:
```

Comment:
sdelete may need a `-accepteula` here to run unattendedly.

Reply:
@rasa thanks, here it is still installed via Chocolatey, which [takes care of that](https://community.chocolatey.org/packages/sdelete#files).

I have just recently switched to manual installation and just [fixed this](https://github.com/gusztavvargadr/packer/commit/e7b0844288abf1d7276586173086c5b51c6bf522#diff-18b3d6447645ed67e7ebc35198b2e861e775cccf4c856cc23a78ded0bb8374e7) on the main branch, but this branch was not synced yet.